scripts/events.js

```diff
@@ -24,6 +23,7 @@ $("button.buttonBack").onclick = () => justBack();
 $("button.buttonStore").onclick = () => goStore();
 $("button.buttonCave").onclick = () => goCave();
 $("button.buttonRestart").onclick = () => {
+  $("button.buttonRestart").classList.add("spin");
```

Comment:
i was thinking about that but i didnt know how to utilize the await reslove function u created because the main perameters are surrounding the text, im guessing i only fill out the timing values? and disregard the text so i can use the timing in the `button.buttonRestart` function.